requester/mctp_endpoint_discovery.cpp

```diff
@@ -35,6 +35,7 @@ MctpDiscovery::MctpDiscovery(sdbusplus::bus_t& bus,
     }
     catch (const std::exception& e)
     {
+        error("Failed to call the D-Bus Method ERROR={ERR}", "ERR", e.what());
```

Comment:
do we want to mention what DBus method failed?

Reply:
yes, in exception case, we will not be able to identify where it's failed so trace is required.

Reply:
yes, I got it but do you want to mention in the error the name of the DBus method?

Reply:
other Dbus exception catching block, method name is not mentioned, so keeping same as other traces.

Reply:
Kamal, rather than putting a generic D-bus error, can you mention something like "D-bus method call to get the managed objects under MCTP Control failed".

Reply:
updated as per comment.